pandas/core/indexes/timedeltas.py

```diff
@@ -299,9 +299,11 @@ def _formatter_func(self):
     def _format_native_types(self, na_rep="NaT", date_format=None, **kwargs):
         from pandas.io.formats.format import Timedelta64Formatter
 
-        return Timedelta64Formatter(
-            values=self, nat_rep=na_rep, justify="all"
-        ).get_result()
+        return np.array(
+            Timedelta64Formatter(
+                values=self, nat_rep=na_rep, justify="all"
+            ).get_result()
+        )
```

Comment:
use asarray or pass copy=False

Reply:
sure. was keeping the code as close to original as possible.